include/dmlc/base.h

```diff
@@ -11,6 +11,12 @@
 #define DMLC_USE_GLOG 0
 #endif
 
+/*!
+ * \def DMLC_USE_LOGGING_LIBRARY
+ * Whether to use user-defined logging library. Change this to an
+ * include path like <my/lib/logging.h> to replace DMLC's builtin logging.
```

Comment:
```suggestion
 * Whether to use user-defined logging library. If defined, dmlc will not define
 * the macros CHECK() and LOG() and instead locate CHECK() and LOG()
 * from the value of DMLC_USE_LOGGING_LIBRARY. The
 * DMLC_USE_LOGGING_LIBRARY macro shall be of form <my_logging.h>.
 * Make sure to define CHECK() and LOG() macros in the provided header;
 * otherwise the build will fail.
```